core/src/main/java/com/oath/oak/Chunk.java

```diff
@@ -76,17 +124,6 @@
 
     /*-------------- Constructors --------------*/
 
-    // static constructor - access and create a new instance of Unsafe
```

Comment:
Currently I do not see why was it removed... Maybe just moved to the other place?

Reply:
I used the unsafe instance from UnsafeUtils instead of creating a new instance.